release_dashboard/lib/widgets/common/dialog_prompt.dart

```diff
@@ -4,37 +4,140 @@
 
 import 'package:flutter/material.dart';
 
-/// Widget that prompts an alert dialogue to the current window and forces the user to choose between two options.
+typedef DynamicFuture = Future<dynamic> Function();
+
+/// Prompts an alert to the current window and forces the user to choose between two options.
+///
+/// The `context` argument is used to look up the [Navigator] for the dialog.
```

Comment:
You should use `[context]` instead of backticks. Backticks indicate runnable code, whereas the square brackets refer to variables

Reply:
It was based off of this: https://github.com/flutter/flutter/blob/23c20d702badf669ac63e1777314da45419f46db/packages/flutter/lib/src/material/dialog.dart#L958

Reply:
https://dart.dev/guides/language/effective-dart/documentation#prefer-backtick-fences-for-code-blocks

Reply:
Done, please take another look.